app/src/panel/models/page/blueprint/pages.php

```diff
@@ -22,7 +22,7 @@ class Pages extends Obj {
   public function __construct($params = array()) {
 
     if($params === true) {
-      $this->template = blueprint::all();
+      $this->template = true;
```

Comment:
`true` is a bit misleading as a general "special value". Maybe this should be a class constant with an integer value (like `self::DETECT_DYNAMICALLY`) to make clear what it means.